sources/AppBundle/Controller/Admin/HealthcheckController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\Controller\Admin;
+
+use Afup\Site\Corporate\_Site_Base_De_Donnees;
+use AppBundle\Event\Model\Repository\EventRepository;
+use CCMBenchmark\TingBundle\Repository\RepositoryFactory;
+use DateTime;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Kernel;
+
+class HealthcheckController extends AbstractController
+{
+    private RepositoryFactory $ting;
+
+    public function __construct(RepositoryFactory $ting)
+    {
+        $this->ting = $ting;
+    }
+
+    public function __invoke(): Response
+    {
+        $php = new DateTime();
+
+        $bdd = new _Site_Base_De_Donnees();
+        $mysqlBdd = $bdd->obtenirUn('SELECT CURRENT_TIMESTAMP');
+        $mysqlBdd = new DateTime($mysqlBdd);
+
+        $repo = $this->ting->get(EventRepository::class);
+        $mysqlTing = $repo->getQuery('SELECT CURRENT_TIMESTAMP')->execute()['CURRENT_TIMESTAMP'];
+        $mysqlTing = new DateTime($mysqlTing);
+
+        $hasDiff = $php->getTimestamp() !== $mysqlBdd->getTimestamp();
+        if (!$hasDiff) {
+            $hasDiff = $php->getTimestamp() !== $mysqlTing->getTimestamp();
+        }
```

Comment:
J'ai pas testé si ça marche comme ça ?

```suggestion
        $hasDiff = $php->getTimestamp() !== $mysqlBdd->getTimestamp() || $php->getTimestamp() !== $mysqlTing->getTimestamp();
```

Reply:
Yes, bien vu